File: android2_lesson03/src/java/java/com/example/ama/android2_lesson03/ui/MainActivity.java
package com.example.ama.android2_lesson03.ui;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

import com.example.ama.android2_lesson03.R;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        if (savedInstanceState == null) {
            Launcher.runSearchFragment(this, false);
        }
    }
}

File: android2_lesson03/src/kotlin/java/com/example/ama/android2_lesson03/ui/MainActivity.kt
package com.example.ama.android2_lesson03.ui

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import com.example.ama.android2_lesson03.R

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        savedInstanceState ?: Launcher.runSearchOnTheMapFragment(this)
    }
}